test/ui/views/grades_details_view_test.dart

```diff
@@ -8,10 +8,8 @@ import 'package:notredame/core/managers/course_repository.dart';
 import 'package:notredame/core/managers/settings_manager.dart';
 
 // MODELS
-import 'package:notredame/core/models/course.dart';
-import 'package:notredame/core/models/course_evaluation.dart';
-import 'package:notredame/core/models/course_summary.dart';
-import 'package:notredame/core/models/evaluation.dart' as model;
+import 'package:ets_api_clients/models.dart';
+import 'package:ets_api_clients/models.dart' as models;
```

Comment:
```suggestion
import 'package:ets_api_clients/models.dart' as models;
```

Reply:
The thing is if I import it only with `as models`  we will have to prefix every model used in this file with `models.`. We only need `models.Evaluation` to be prefix as it cause a duplicate entry from another package.

One thing I could do would be to rename the model Evaluation to something different (maybe `CourseEvaluation`) so that it isn't conflicting with another import. This would occasion many rename in a lot of files so I opted for another less "cleaner" approach which import models globally and import models with a prefix models.

Maybe in another PR we could change the behavior of the package. For this PR, I just want to migrate all the stuff and achieve the stability we had before without altering how it worked 😄.

Reply:
Right....forgot that XD Yeah I agree with you renaming Evaluation to CourseEvaluation could be a good idea

Reply:
I didn't see before, but there already is a CourseEvaluation. do you have any idea about other names I could used?

Reply:
Oh, the current `CourseEvaluation` is more a review because it's the student that rate/review the course. Here is my proposition:
`CourseEvaluation` is renamed `CourseReview` being more accurate and `Evaluation` renamed in `CourseEvaluation` or `CourseExam`. What do you think?

Reply:
This is perfect thanks! I'll change that right away 😄